### DISCUSSION TITLE
After 3.4.8 getting content configuration warning that seems wrong?

### DISCUSSION THREAD

**Discussion by OliverGrimsley** [tailwindlabs/tailwindcss#14148] [Help] [answered]
After installing v 3.4.8, I get: 
```
warn - Your `content` configuration includes a pattern which looks like it's accidentally matching all of `vendor` and can cause serious performance issues.
warn - Pattern: `./resources/views/**/*.php`

```
Is this accurate? I only have a few folders and a few blade files in that pattern match. Seems like this warning is spurious?

**Comment by philipp-spiess** [comment, marked as answer]
Hey @OliverGrimsley! Sorry for the troubles there. The warning in 3.4.8 is definitely firing more often than we want it to. There's another discussion for this open already so I’m going to link to it and close this one but expect a fix out for this very soon: https://github.com/tailwindlabs/tailwindcss/discussions/14145
